setup.py

```diff
@@ -16,7 +16,7 @@
 
 setup(
     name='esss_fix_format',
-    version='2.1.2',
+    version='3.0.0',
```

Comment:
wouldn't it be better to use ``setuptools_scm`` than manually change every time the version on the ``setup.py``?

Reply:
Sure, I created issue #56 to tackle this ;)